test/features/support/oidc_userinfo.js

```diff
@@ -0,0 +1,113 @@
+const { stash, spec } = require("pactum");
+const { regex } = require("pactum-matchers");
+const { Given, When, Then, Before, After } = require("@cucumber/cucumber");
+const { localhost } = require("./helpers/helpers");
+
+const baseUrl = `${localhost}oidc/userinfo`;
+
+const jsonToBase64 = (jsonObj) => {
+  const jsonString = JSON.stringify(jsonObj);
+  return Buffer.from(jsonString).toString("base64");
+};
```

Comment:
In PR https://github.com/GovStackWorkingGroup/bb-identity/pull/12/files you are wrote the same function. Please move it to the shared file, for example `utils.js` file in `helpers` folder.